sx-auth.el

```diff
@@ -38,14 +38,16 @@
   "http://vermiculus.github.io/sx.el/auth/auth.htm")
 (defconst sx-auth-client-id
   "3291")
-(defvar sx-auth-access-token
-  nil
+
+(defstash sx-auth-access-token
+    "auth.el" sx nil
```

Comment:
Given how catastrophic it would be to have two stashes with the same file name, maybe we should use more specific names for future stashes. Or (even better) maybe stash.el could take care of choosing unique file names based on variable name.